src/cosmosdb-preview/azext_cosmosdb_preview/vendored_sdks/azure_mgmt_cosmosdb/models/_cosmos_db_management_client_enums.py

```diff
@@ -179,8 +179,8 @@ class PrimaryAggregationType(str, Enum):
     last = "Last"
 
 
-class BackupType(str, Enum):
-
+class BackupPolicyType(str, Enum):
```

Comment:
@kavskalyan by any chance do you know why was this called BackupType ? It should have been BackupPolicyType

Reply:
I did not notice it too. 
I verified it in the stable version, but did not check in the preview version, since we added the BackupPolicyType explicitly in the code. 
But its good to be fixed. I'll start using this enum in my next PR.